extern/ttconv/truetype.h

```diff
@@ -41,6 +41,13 @@ typedef struct
     unsigned short int fraction;
     } Fixed;
 
+/* A 2.14 fixed point number */
+typedef struct
+    {
+    SHORT whole;
+    USHORT fraction;
+    } F2DOT14;
```

Comment:
This is basically the same as `Fixed`, except using the Microsoft type names. I don't think we should use those, as the comment says they are 'types used in TrueType font files', but this structure is for ttconv use. Simpler could be to just do: `typedef Fixed F2DOT14;`

Reply:
Sure, I'll change this